.github/workflows/docker-integration-test.yaml

```diff
@@ -1,24 +1,27 @@
 name: docker-integration-test
-on: [push]
+on: [pull_request, push]
```

Comment:
@mrchrisadams Push doesn't work for forks 🤦‍♂️ now we will run the integration tests once a PR is created.

I also did some code gardening and updated the actions versions.